### PR TITLE
Adding funding ideas to the README

### PR DESCRIPTION
**Pull request by datapythonista**

### CODE REVIEW

**Review by dujm — APPROVED** [review]

File: README.md
Comment (sara-02) [inline README.md:31]: WDYT about Rails Summer of Code?
https://railsgirlssummerofcode.org/

Comment (datapythonista) [reply README.md:31]: for the name sounds like working on pandas will be out of scope, but if it's not, happy to get it in the list in a follow up PR

Comment (sara-02) [reply README.md:31]: Well, in the past they have accepted projects in all languages and not just Rails.
But, for the time being they are ungoing a revamp. Maybe we could revisit them later.
https://railsgirlssummerofcode.org/blog/2019-03-21-the-future-of-rgsoc

Comment (datapythonista) [reply README.md:31]: That's good to know, then surely worth having it. Do you mind creating an issue to add it as a follow up once this is merged?

**Comment by sara-02** [comment]
@datapythonista Are we waiting for any updates on the PR, or can it be merged?

**Comment by datapythonista** [comment]
I think it should be good to be merged.

**Comment by sara-02** [comment]
Thanks, @datapythonista and @dujm  merging this one.

**Pull request merged**
